app/src/cli.c

```diff
@@ -431,6 +466,7 @@ scrcpy_parse_args(struct scrcpy_cli_args *args, int argc, char *argv[]) {
         {"window-height",          required_argument, NULL, OPT_WINDOW_HEIGHT},
         {"window-borderless",      no_argument,       NULL,
                                                   OPT_WINDOW_BORDERLESS},
+        {"codec-profile",          required_argument, NULL, 'P'},
```

Comment:
This is an advanced option, I would prefer not to consume a "short" option (`-P`) for that.

Reply:
Yes that's totally fine, at first i had a bit of a problem understanding the code so i didn't know if it will work without a short version of the option, its ok to remove it -P and stay with --codec-profile only